.pipelines/pipeline.yaml

```diff
@@ -14,7 +14,7 @@ pr:
 trigger:
   branches:
     include:
-      - gh-readonly-queue/master/*
+      - gh-readonly-queue/release/v1.5/*
```

Comment:
maybe we should preemptively go
```suggestion
      - gh-readonly-queue/release/*
```
so that we don't have to add to this every few months?

Reply:
Should this target master instead and then backport?

Reply:
We can take this and make that change an master also